hivemind/moe/server/server.py

```diff
@@ -344,15 +331,15 @@ def _server_runner(pipe, *args, **kwargs):
         return
 
     try:
-        dht_maddrs = server.dht.get_visible_maddrs() if server.dht is not None else None
-        pipe.send((True, (server.listen_on, dht_maddrs)))
+        dht_maddrs = server.dht.get_visible_maddrs()
+        pipe.send((True, (server.dht.peer_id, dht_maddrs)))
         pipe.recv()  # wait for shutdown signal
 
     finally:
         logger.info("Shutting down server...")
         server.shutdown()
         server.join()
-        logger.info("Server shut down")
+        logger.info("Server shut down.")
```

Comment:
nit: We usually don't use trailing dots in the log messages (e.g., see #419). Please remove them.